packages/pymedphys_dicom/src/pymedphys_dicom/_install_requires.py

```diff
@@ -0,0 +1,8 @@
+install_requires = [
+    'pymedphys_analysis',
+    'pymedphys_utilities',
+    'numpy < 1.16, >= 1.12',
```

Comment:
How important is it that we keep these pins consistent across packages? If so, can we somehow apply these pins "DRYly"? Or perhaps include a test that every pinned external dependency has the same pins across all subpackages?

Reply:
It's very important to be DRY for this and a range of other things. The aim
is to have these dependency pins templated and then propegated throughout
via a template. It will likely happen before every commit using a commit
hook. Or at a point similar to that...

My plan was to build a dependency tree from the files themselves, have a
reference set of pins in one location, then using the dependency tree
propegate the dependencies to each library with pins that match the
reference.